### PR TITLE
Update QuarkusPluginExtension.java

### PR DESCRIPTION
**Pull request by espressoo0**

### CODE REVIEW

**Comment by maxandersen** [comment]
what is this change for?

**Comment by zestzoe** [comment]
> what is this change for?

The purpose of this code change was to address a warning in the existing codebase and introduce a new method that allows retrieval of the last file from a given file collection.
- In the original code, it was observed that the method `getLastFile` lacked a documentation element : the @param Javadoc tag for the `fileCollection` parameter. 
- this resulted in a warning during the build process. To rectify this, the change incorporates the necessary Javadoc tag
- Regarding the implementation of the method itself, it employs a simple loop structure to iterate over the elements of the `fileCollection`. With each iteration, the `lastFile` variable is assigned the current file, effectively tracking the most recent file encountered. Once the loop concludes, the `lastFile` variable holds the final file in the collection. In scenarios where the file collection is empty, the method gracefully returns null as an indication that no files were found.

File: devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java [hunk 1]
```diff
@@ -136,10 +136,18 @@ public Property<Boolean> getCacheLargeArtifacts() {
...
-
-    public String finalName() {
-        return getFinalName().get();
-    }
+    
+     /**
+     * @param fileCollection The file collection from which to retrieve the last file.
+     * @return The last file in the file collection, or null if the file collection is empty.
+     */
+    public static File getLastFile(FileCollection fileCollection) {
```
Comment (gsmet) [inline devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java:144]: I read your answer to Max but I still don't understand why this change is needed? Why do you need this method?
